docs/source/guides/reversing/squirrelreversing.rst

```diff
@@ -0,0 +1,112 @@
+Pushing Values to the SQVM Stack
+================================
+
+Before starting to reverse engineer how to push values to the Squirrel stack, it's recommended to read the `Squirrel documentation
+<http://www.squirrel-lang.org/squirreldoc/index.html>`_, especially the `Embedding-
+<http://www.squirrel-lang.org/squirreldoc/reference/embedding_squirrel.html>`_ and `API Reference
+<http://www.squirrel-lang.org/squirreldoc/reference/api_reference.html>`_.
+
+A lot of Respawn's fork of Squirrel is very similar to Squirrel3, especially the Squirrel API functions.
+
+Setup
+-----
+
+Before you can start reverse engineering you need to install software to disassemble and decompile the binaries. This guide will use `Ghidra
```

Comment:
We probably also mention IDA. Especially for hooking functions (which is not the goal of this page but whatever) IDA is preferred as to my knowledge, the reversed code it generates compiles closer to the original code compared to Ghidra.